### PR TITLE
Update footer links to March 2021

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
Looks good to me.

File: client/i18n/locales/english/trending.json [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
-  "article27link": "https://www.freecodecamp.org/news/windows-10-start-menu-not-working-solved/",
-  "article28title": "Windows 10 Command Line",
-  "article28link": "https://www.freecodecamp.org/news/how-to-open-the-command-prompt-in-windows-10/",
-  "article29title": "Google Account Recovery",
-  "article29link": "https://www.freecodecamp.org/news/google-account-recovery-change-gmail-password/"
+  "article0title":"10 to the Power of 0",
+  "article0link":"https://www.freecodecamp.org/news/10-to-the-power-of-0-the-zero-exponent-rule-and-the-power-of-zero-explained/",
+  "article1title":"Git Reset to Remote",
+  "article1link":"https://www.freecodecamp.org/news/git-reset-to-remote-head-how-to-reset-a-remote-branch-to-origin/",
+  "article2title":"r value statistics",
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/trending.json:6]:
```suggestion
  "article2title":"R Value in Statistics",
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
To keep consitency - uppercase first letter.

Otherwise, I commonly see the R value represented as a capital, anyway.

File: client/i18n/locales/english/trending.json (hunk 1 above)
Comment (ahmadabdolsaheb) [reply client/i18n/locales/english/trending.json:6]: @scissorsneedfoodtoo, should we go the capital version? the following is what was on he card.
Screen Shot 2021-03-30 at 11 05 30 AM

Comment (scissorsneedfoodtoo) [reply client/i18n/locales/english/trending.json:6]: Thanks for catching this @ShaunSHamilton. I think we did miss the title casing on the Trello card for this article.

I did a bit of searching and have also seen it as either R or r in other titles. Let's go with this suggestion to keep the casing in the footer consistent.

File: client/i18n/locales/english/trending.json [hunk 2]
```diff
@@ -1,62 +1,62 @@
...
+  "article6link":"https://www.freecodecamp.org/news/model-view-architecture/",
+  "article7title":"React Testing Library",
+  "article7link":"https://www.freecodecamp.org/news/react-testing-library-tutorial-javascript-example-code/",
+  "article8title":"ASCII Table Chart",
+  "article8link":"https://www.freecodecamp.org/news/ascii-table-hex-to-ascii-value-character-code-chart-2/",
+  "article9title":"Data Validation",
+  "article9link":"https://www.freecodecamp.org/news/form-validation-with-html5-and-javascript/",
+  "article10title":"Recursion",
+  "article10link":"https://www.freecodecamp.org/news/what-is-recursion-in-javascript/",
+  "article11title":"ISO file",
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/trending.json:24]:
```suggestion
  "article11title":"ISO File",
```

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Pull request merged**